R/fit.R

```diff
@@ -705,17 +702,17 @@ CmdStanMCMC <- R6::R6Class(
       } else {
         if (self$runset$args$validate_csv) {
           fixed_param <- runset$args$method_args$fixed_param
-          data_csv <- read_cmdstan_csv(
+          csv_contents <- read_cmdstan_csv(
```

Comment:
This and every other change to `csv_contents` isn't important, it just makes naming consistent throughout our code